src/Uno.UI.RemoteControl.DevServer.Tests/Helpers/SolutionHelper.cs

```diff
@@ -127,9 +137,26 @@ public void EnsureUnoTemplatesInstalled()
 		}
 	}
 
+	public async Task ShowDotnetVersionAsync()
+	{
+		var startInfo = new ProcessStartInfo
+		{
+			FileName = "dotnet",
+			Arguments = "--info",
+			RedirectStandardOutput = true,
+			RedirectStandardError = true,
+			UseShellExecute = false,
+			CreateNoWindow = true,
+			WorkingDirectory = _tempFolder,
+		};
+
+		var (exitCode, output) = await ProcessUtil.RunProcessAsync(startInfo);
```

Comment:
Are tests in this assembly parallelized?
How many tests do start processes using this utility? This utility exhausts the threadpool.